crates/polars-ops/src/series/ops/business.rs

```diff
@@ -269,7 +271,7 @@ fn add_business_days_impl(
     mut n: i32,
     week_mask: &[bool; 7],
     n_business_days_in_week_mask: i32,
-    holidays: &[i32],
+    holidays: &[i32], // Caller's responsibility to ensure it's sorted.
```

Comment:
We have a `check_sorted`. I think we should verify.

Reply:
are you sure? this function `add_business_days_impl` is called with a hot for-loop, and is only called from within this file, only from `add_business_days`, and only after `holidays` has been sorted

this function is also hypothesis-tested against NumPy's implementation, with holidays randomly generated in random order

Happy to call `check_sorted` if you prefer of course, just thought I'd explain why IMO it's safe enough to not do so

Reply:
Ah,  no, not in a hot-loop :see_no_evil: 

Just when we enter Polars from a user perspective. I misread "Caller's" as "User's"`.  Nvm.